pkg/platform/baremetal/baremetal.go

```diff
@@ -110,6 +111,28 @@ func (c *config) Initialize(ex *terraform.Executor) error {
 		return err
 	}
 
+	// TODO: A transient change which shall be reverted in a follow up PR to handle
+	// https://github.com/kinvolk/lokomotive/issues/716.
+	// Extract control plane chart files to cluster assets directory.
+	for _, c := range platform.CommonControlPlaneCharts {
+		src := filepath.Join(assets.ControlPlaneSource, c)
+		dst := filepath.Join(assetDir, "cluster-assets", "charts", "kube-system", c)
```

Comment:
With #704, not all system charts will be installed in `kube-system` namespace. How can we handle it then?

Reply:
We can put subdirectories under `assets/charts/control-plane` which represent the namespace and include the subdirectory in the asset path either by convention or by configuration.

Reply:
Ended up handling this by configuration: I've added the following type:

```go
// ControlPlaneChart represents a Helm chart belonging to a Lokomotive control plane component.
type ControlPlaneChart struct {
	// The name of the chart.
	Name string
	// The namespace into which the chart should be deployed.
	Namespace string
}
```

And then:

```go
// CommonControlPlaneCharts defines a list of control plane Helm charts to be deployed for all
// platforms.
var CommonControlPlaneCharts = []ControlPlaneChart{
	{
		Name:      "calico",
		Namespace: "kube-system",
	},
	{
		Name:      "kube-apiserver",
		Namespace: "kube-system",
	},
	{
		Name:      "kubernetes",
		Namespace: "kube-system",
	},
	{
		Name:      "pod-checkpointer",
		Namespace: "kube-system",
	},
	{
		Name:      "lokomotive",
		Namespace: "lokomotive-system",
	},
}
```

It's simple and it works.